pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -123,6 +123,11 @@ message MessageMetadata {
     // internal metadata instead of application published data.
     // Markers will generally not be propagated back to clients
     optional int32 marker_type = 20;
+
+	// transaction related message info
```

Comment:
`marker_type` is using spaces, while other fields are using `tabs`. I would suggest fixing all the tabs to spaces.

Reply:
Ok.